ynr/apps/official_documents/views.py

```diff
@@ -37,6 +38,16 @@ def get_context_data(self, **kwargs):
         context["post_label"] = post.label
         return context
 
+    def form_valid(self, form):
+        LoggedAction.objects.create(
+            user=self.request.user,
+            post_election=form.instance.post_election,
+            action_type="person-update",
```

Comment:
This is a bug. Creating the logged action as a `person-update` with no `person_id` causes everything to start throwing `Reverse for 'person-view' with keyword arguments '{'person_id': ''}'` once its in the history